app/javascript/shared/components/tags.jsx

```diff
@@ -74,6 +74,7 @@ export class Tags extends Component {
       cursorIdx: 0,
       prevLen: 0,
       showingRulesForTag: null,
+      showingTopTags: false,
```

Comment:
naming (nitpick non-blocking): `showingRulesForTag` is already like this, but consider `showTopTags`.